app/pages/resource/resource-info/ResourceInfo.spec.js

```diff
@@ -55,16 +60,48 @@ describe('pages/resource/resource-info/ResourceInfo', () => {
       name: 'Unit name',
       streetAddress: 'Test street 12',
     });
-    const address = getWrapper({ unit }).find('.address');
+    const address = getWrapper({ unit }).find('.app-ResourceInfo__address');
     const expected = 'Unit name, Test street 12, 99999 Helsinki';
 
     expect(address.props().children).to.equal(expected);
   });
 
-  it('renders ResourceIcons component', () => {
-    const resourceIcons = getWrapper().find(ResourceIcons);
+  it('renders a hourly price', () => {
+    const hourlyPriceSpan = getWrapper().find(
+      '.app-ResourceInfo__hourly-price'
+    );
+
+    expect(hourlyPriceSpan.is('span')).to.be.true;
+    expect(hourlyPriceSpan.text()).to.contain('30 €/h');
+  });
+
+
+  it('renders a label with people capacity', () => {
+    const peopleCapacityLabel = getWrapper().find(
+      '.app-ResourceInfo__peopleCapacity'
+    );
+
+    expect(peopleCapacityLabel.is(Label)).to.be.true;
+    expect(peopleCapacityLabel.prop('shape')).to.equal('circle');
+    expect(peopleCapacityLabel.prop('size')).to.equal('medium');
+    expect(peopleCapacityLabel.prop('theme')).to.equal('orange');
+    expect(peopleCapacityLabel.children().at(0).text()).to.equal('16');
```

Comment:
- [ ] Having tests that rely this deeply on the html structure of the component is a bit problematic. Just removing the space between the icon and text will break the tests. Or switching the icon and text around.
I think it's enough to check that the value we are interested get's rendered. So here would be enough to check that label children contain resource.peopleCapacity. The hardcoded value '16' is also a bit problematic. Is the value always '16' or just because the defaultProps.resource.peopleCapacity is 16. The test doesn't really tell that. 

So testing the children could be something like:
```javascript
const expectedValue = defaultProps.resource.peopleCapacity;
expect(peopleCapacityLabel.children().contains(expectedValue)).to.be.true;
```